### ISSUE TITLE
Navigation elements should be visible at all times, not fade out

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/nextcloud-vue#2811]

**Comment by JuliaKirschenheuter** [comment]
@AndyScherzinger do we still have that places, where navigation elements are not visible?

**Comment by PVince81** [comment]
I think this was in the context of modal dialogs but can't remember which

the only piece I remember is that the dialog title used to be outside the dialog, at the top of the window and the cross icon for closing would fade out

**Comment by PVince81** [comment]
it might also have been about the next/previous buttons: https://github.com/nextcloud/nextcloud-vue/pull/2695

**Comment by PVince81** [comment]
and looking further I suspect that the prev/next buttons were intended to be used by the viewer app for the picture slideshow.
picture slideshows usually would hide the buttons to let the user see the picture better

**Comment by PVince81** [comment]
I've tested the modal and the prev/next buttons never disappear

**Comment by PVince81** [comment]
while testing, I discovered that the button's contrast was not that good, raised https://github.com/nextcloud/nextcloud-vue/issues/2954
